salt/returners/zabbix_return.py

```diff
@@ -15,20 +15,34 @@
 '''
 from __future__ import absolute_import
 import logging
+import os
 log = logging.getLogger(__name__)
 
 
+if os.path.exists('/usr/local/zabbix/bin/zabbix_sender') and os.path.exists('/usr/local/zabbix/etc/zabbix_agentd.conf'):
+    zabbix_sender = '/usr/local/zabbix/bin/zabbix_sender'
+    zabbix_config = '/usr/local/zabbix/etc/zabbix_agentd.conf'
+    HAS_ZABBIX = True
+elif os.path.exists('/usr/bin/zabbix_sender') and os.path.exists('/etc/zabbix/zabbix_agentd.conf'):
+    zabbix_sender = '/usr/bin/zabbix_sender'
+    zabbix_config = '/etc/zabbix/zabbix_agentd.conf'
+    HAS_ZABBIX = True
+else:
+    log.warning('Zabbix returner: No zabbix_sender and zabbix_agend.conf found.')
```

Comment:
Instead of logging this as a warning, the loader supports returning a tuple consisting of a boolean value and a message, like this:
```
else:
    return False, 'Zabbix returner: No zabbix_sender and zabbix_agend.conf found.'
```